assembl/models/auth.py

```diff
@@ -685,6 +690,40 @@ def __init__(self, **kwargs):
 
         super(User, self).__init__(**kwargs)
 
+    @classmethod
+    def populate_db(cls, db=None):
+        default_config = config.get_config()
+        # retrieve the configured machines
+        machines_data = default_config.get('machine.machines', '')
+        if machines_data:
+            from ..auth.util import add_user
+            db = db or cls.default_db()
+            # retrieve existing machines
+            machines = db.query(cls).filter(
+                cls.is_machine == True  # noqa: E712
+                )
+            # retrieve machines ids
+            all_machines_ids = [machine.username.username for machine in machines]
+            machines_data = [machine.split(',') for machine in machines_data.split('/')]
+            machines_data = {machine[0].strip(): {'name': machine[1].strip(), 'password': machine[2].strip()}
+                             for machine in machines_data}
+            # retrieve the not existing machines
+            machines_ids = [id for id in machines_data if id not in all_machines_ids]
+            # add the not existing machines
+            for machine_id in machines_ids:
+                machine = machines_data[machine_id]
+                try:
+                    add_user(
+                        machine.get('name'),
+                        None,
+                        machine.get('password'),
+                        R_CATCHER,
+                        username=machine_id,
+                        is_machine=True
+                    )
+                except Exception:
+                    continue
```

Comment:
You catch the exception in the case the user already exists? This may hide other errors... You should check if the user exists first. With `session.query(Username).filter_by(username=machine_id).count())` for example.